diff -crN bell_yespower_python3.orig/setup.py bell_yespower_python3/setup.py
*** bell_yespower_python3.orig/setup.py Mon Jul 15 00:54:29 2019
--- bell_yespower_python3/setup.py      Mon Jul 15 00:55:03 2019
***************
*** 6,12 ****
                                         'yespower-opt.c',
                                         'sha256.c'
                                         ],
!                             extra_compile_args=['-O2', '-funroll-loops', '-fomit-frame-pointer'],
                              include_dirs=['.'])

  setup (name = 'bell_yespower',
--- 6,12 ----
                                         'yespower-opt.c',
                                         'sha256.c'
                                         ],
!                             extra_compile_args=['-O2'],
                              include_dirs=['.'])

  setup (name = 'bell_yespower',